### DISCUSSION TITLE
编译结果

### DISCUSSION THREAD

**Discussion by postfaxcn** [vlang/v#15392] [General]
同样一个hellow_word.v源码，在不同的v语言版本上，编译出来的结果，还真差不少字节。目前有优化参数吗？

**Comment by postfaxcn** [comment]
同时v执行程序也差好多字节

**Comment by postfaxcn** [comment]
同样的源码在linux与mac上编译windows结果也是字节数差的比较多

**Comment by tzSharing** [comment]
```
>v -prod hellow_word.v
>hellow_word.exe
```
更多可选参数可通过`v help` 或 `v help build`寻求帮助
